test/test_variant.py

```diff
@@ -98,3 +101,38 @@ def test_deletion_no_suffix():
     eq_(variant.start, 10)
     eq_(variant.end, 11)
     eq_(variant.short_description, "chr1 g.10_11delAA")
+
+def assert_variants_completely_equal(variant1, variant2):
+    eq_(variant1, variant2)
```

Comment:
Should something like `completely_equal` or `equal_with_info` be a member of `Variant`?